File: src/main/java/com/kairlec/service/MimeMapService.java
package com.kairlec.service;

import com.kairlec.pojo.MimeMap;
import org.springframework.stereotype.Service;

import java.util.List;

@Service
public interface MimeMapService {

    Integer init();

    List<MimeMap> getAll();

    MimeMap getMimeMap(String ext);

    List<MimeMap> getSection(List<String> exts);

    Integer insertMimeMap(MimeMap mimeMap);

    Integer deleteMimeMap(String ext);

    Integer updateMimeMap(MimeMap mimeMap);
}

File: src/main/java/com/kairlec/service/MimeMapService.kt
package com.kairlec.service

import com.kairlec.pojo.MimeMap
import org.springframework.stereotype.Service

@Service
interface MimeMapService {
    fun init(): Int?
    fun all(): List<MimeMap>?
    fun getMimeMap(ext: String): MimeMap?
    fun getSection(exts: List<String>): List<MimeMap>?
    fun insertMimeMap(mimeMap: MimeMap): Int?
    fun deleteMimeMap(ext: String): Int?
    fun updateMimeMap(mimeMap: MimeMap): Int?
}
